src/ad-sizes.ts

```diff
@@ -38,6 +38,13 @@ class AdSize extends Array<number> {
 		return [this[0], this[1]];
 	}
 
+	// The inserted advert is a fixed size within its slot.
+	// Examples that are not a fixed size are: fluid ads, empty or
+	// out of page ads, and merch ads in a [88, ??] size format.
+	public isFixedSize(): boolean {
```

Comment:
Is there a better name for this? I want to differentiate between the ad sizes that expect an advert to load normally within the slot and are constrained by the size, and adverts that can expand out of the slot or are dummy sizes.

Reply:
This is tricky naming! Some options that spring to mind:

positive: `Real`, `Actual`,  `Numeric` 
negative: `Proxy`, `Symbol`, `Token`

I think `isNumeric` / `isProxy` are my favourites so far but interested in what others think.

Reply:
Could use the term `flexible` for fluid, merch etc? then `isFlexible` and `!isFlexible()`

Reply:
Thanks both for your suggestions! I used fixedSize as I found that phrase is the Google definition of fluid ads.

I like isProxy the best as it more closely describes the size of the ad as opposed to the advert so I'll go with that.

OR...should this live on the Advert class in Frontend? Is this so hard to name because it doesn't belong in this AdSize class?

Reply:
I'd be in favour of keeping the logic here for now rather than Frontend.

But it does feel like `AdSize` is mixing concerns between storing a single size and then knowing about 'other' sizes. We can keep as is for now and follow up with a refactor?

Reply:
Yeah I agree with that there is a mixing of concerns that could use a refactor. 

Also, in frontend, we let the size equal the string "fluid", which isn't quite right because AdSize already handles the case of a fluid ad:
```
class Advert {
...
size: AdSize | 'fluid' | null = null;
...
}
```